Projects/hCoreUI/Sources/hForm/hForm.swift

```diff
@@ -174,3 +218,20 @@ extension View {
         self.environment(\.hUseNewStyle, true)
     }
 }
+
+private struct EnvironmentHUseBlur: EnvironmentKey {
+    static let defaultValue = false
+}
+
+extension EnvironmentValues {
+    public var hUseBlur: Bool {
+        get { self[EnvironmentHUseBlur.self] }
+        set { self[EnvironmentHUseBlur.self] = newValue }
+    }
+}
+
+extension View {
+    public var hUseBlur: some View {
```

Comment:
We should be able to use hUseBlur on every type of style we want to use. 
We need to implement it in the PresenationOptions in Presentation Library